int16 size
rtabmap_ros/Point3f[] kpts3DVec
